addons/captives/functions/fnc_setSurrendered.sqf

```diff
@@ -42,6 +42,10 @@ if (_state) then {
     if ((vehicle _unit) != _unit) exitWith {WARNING("Cannot surrender while mounted");};
     if (_unit getVariable [QGVAR(isHandcuffed), false]) exitWith {WARNING("Cannot surrender while handcuffed");};
 
+    if (isClass (configFile >> "CfgPatches" >> "ACE_HitReactions")) then {
```

Comment:
What if they have multiple weapons?

Reply:
Throw only the one they currently are holding.

Reply:
Config setting? might not always want to throw your gun away when surrendering.
If you have your rifle on a sling you're not throwing that away.
And if you are holding a pistol, you would probably holster it when surrendering, instead of throwing it away.

Reply:
I'd like to see a per unit setting if weapon is slinged or not